### ISSUE TITLE
add `--pull always` to docker compose command

### ISSUE THREAD

**Issue by gibso** [testcontainers/testcontainers-node#529]
It would be very useful to add a Pull Policy like `--pull always` to the docker compose environment. There is a way to add this parameter to single containers with 
```
const { GenericContainer, AlwaysPullPolicy } = require("testcontainers");

const container = await GenericContainer
  .withPullPolicy(new AlwaysPullPolicy())
```
but I can't see any way to add this to a Docker Compose Environment instance. The argument `--pull` exists in the [Docker API](https://docs.docker.com/engine/reference/commandline/compose_up/#options).

**Expected Behaviour**
there is a way to adjust the --pull parameter for the docker compose up command.

**Actual Behaviour**
there is no way to adjust the --pull parameter for the docker compose up command.

**Comment by cristianrgreco** [comment]
Thanks for raising @gibso. The `--pull` flag is only supported in docker-compose v2, so instead I've implemented an explicit call to `docker-compose pull <services>`, conditional to the pull policy, which works for all versions.

**Comment by cristianrgreco** [comment]
@gibso Released in [9.5.0](https://github.com/testcontainers/testcontainers-node/releases/tag/v9.5.0).
